config/config.go

```diff
@@ -194,7 +194,7 @@ func WriteCurrentContextToConfigFile(value string) error {
 	return nil
 }
 
-func decodeYAMLFile(path string) (map[string]string, error) {
+func DecodeYAMLFile(path string) (map[string]string, error) {
```

Comment:
instead of making this function exportable in this particular package would it makes sense to put that somewhere in utils?